src/conn/conn_capacity.c

```diff
@@ -34,8 +34,28 @@ __capacity_config(WT_SESSION_IMPL *session, const char *cfg[])
     uint64_t chunkcache, total;
 
     conn = S2C(session);
-    chunkcache = total = 0;
+    cap = &conn->capacity;
 
+    WT_RET(__wt_config_gets(session, cfg, "io_capacity.total", &cval));
```

Comment:
I think there's a merge that went wrong here - there's already some checks below for `io_capacity.total`.